src/index.css

```diff
@@ -11,3 +11,33 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.avatar-hover {
+  transition: all 150ms ease-in-out 0s;
+}
+
+.avatar-hover:first-child {
+  margin-right: 0.2rem;
+}
+
+.avatar-hover:last-child {
+  margin-left: 0.2rem;
+}
+
+.avatar-hover:not(:first-child):not(:last-child) {
+  margin: 0 0.2rem;
+}
+
+.avatar-hover:hover {
+  -webkit-box-shadow: 0px 8px 20px -9px #000000;
+  box-shadow: 0px 8px 20px -9px #000000;
+  transform: translateY(-5px);
+}
+
+.p-button {
+  transition: transform 0.33s !important;
+}
+
+.p-button:hover {
```

Comment:
please can you check for existing/duplicate styles https://github.com/EddieHubCommunity/LinkFree/blob/main/src/Components/Links.css#L5